src/types.ts

```diff
@@ -765,19 +765,37 @@ export type PageResponse<T> = ListResponse<T> & {
 }
 
 export type IdentityResponse = {
+  /** Primary identifier of the entity */
   id: string
+  /** Primary identifier of the Item */
   itemId: string
-  birthDate?: Date
-  taxNumber?: string
-  document?: string
-  documentType?: string
-  jobTitle?: string
-  fullName?: string
-  phoneNumbers?: PhoneNumber[]
-  emails?: Email[]
-  addresses?: Address[]
-  relations?: IdentityRelation[]
+  /** Date of birth of the owner */
+  birthDate: Date | null
```

Comment:
is this being mapped to Date, or just string?

Reply:
Ou! Nice catch, it is not being mapped to Date is just the ISO string, we could map it to not change the type definition, what do you think?

Reply:
Yes we could map it like we already do for most of the rest of fields